internal/manager/config.go

```diff
@@ -216,6 +217,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.IngressClassName, "ingress-class", annotations.DefaultIngressClass, `Name of the ingress class to route through this controller.`)
 	flagSet.StringVar(&c.LeaderElectionID, "election-id", "5b374a9e.konghq.com", `Election id to use for status update.`)
 	flagSet.StringVar(&c.LeaderElectionNamespace, "election-namespace", "", `Leader election namespace to use when running outside a cluster.`)
+	flagSet.StringVar(&c.LeaderElectionForce, "force-leader-election", "", `Set to "enabled" or "disabled" to force a leader election behavior. Behavior is normally determined automatically from other settings.`)
```

Comment:
Is this flag `--force-leader-election` for debugging in integration tests only? I think we should not import such a flag, since it is part of our user interface. 
If we really need such a setting, please create another PR and introduce the context.

Reply:
I think it is OK if the setting is really required for debugging. But more notations on the flag to tell that it is for debugging only needs to be added.

Reply:
When the flag is hidden, we skip it in the generated `docs/cli-arguments.md` that's later copied over to docs.konghq.com, I think that's enough. If someone is so desperate to look into the source code looking for a specific flag, they'll see it's hidden - doesn't seem to me like something that should be used on prod 😅

Reply:
Yeah, for weird override switches I'm on the side of labeling it as what actually does and omitting any why's (those can go in the comments for places that use it). Most users follow the default path and aren't going to toggle oddball things out of curiosity. The second "normally" sentence is there as sort of a "you wanna break glass, break glass" ward.